### DISCUSSION TITLE
Configuring the default reserved storage for disks

### DISCUSSION THREAD

**Discussion by chintal** [longhorn/longhorn#4667] [Q&A] [answered]
How can I configure the default "Reserved Storage"? I'm using dedicated block storage (OCI) and don't need to reserve any storage, yet longhorn is applying a 14.7G reservation on a 50G block storage mounted using fstab. I can't seem to find a way to prevent this save for going into each node on the UI and manually changing the reservation. I've tried  `--set defaultSettings.storageMinimalAvailablePercentage=1` during the helm install, but it seems to have had no effect. (Though the corresponding setting on the Longhorn UI settings page has correctly applied.)

Is there a configuration option for longhorn or a node label I can apply to automate this?

I'm labelling my storage nodes as `createDefaultDiskLabeledNodes=true`, and intend to pre-mount appropriate block storage instances on those nodes during initial VM/instance provisioning, before the node is added to the cluster.

**Comment by c3y1huang** [comment, marked as answer]
Ref https://longhorn.io/docs/1.3.1/advanced-resources/default-disk-and-node-config/#customizing-default-disks-for-new-nodes

And you can also customize existing nodes via UI: Node > Operation > Edit node and disks.
![Screenshot-2022-10-04-10:38:13](https://user-images.githubusercontent.com/22583541/193722750-9cc6724b-64ed-4ce7-9bad-11d457d0ff9b.png)

**Reply by chintal** [reply]
Thanks! Not sure how I missed that.
